test/rules/file-header/empty-file/test.js.lint

```diff
@@ -1 +1,2 @@
-// empty.
```

Comment:
Previously this file had just a single-line comment. That case is now covered by both the `good-single-line` and `bad-single-line` tests. I have modified this file to truly be empty.